### DISCUSSION TITLE
issues : about extra params

### DISCUSSION THREAD

**Discussion by LancerMar** [LancerMar/ProjectDemo#3] [General]
why we use "int frame" as params in "App::fftData(int32_t* buffer, int frames)", it is not used ? maybe we should delete it?

```
/*
 * fft process data
 */
void App::fftData(int32_t* buffer, int frames) {
    fft->fill_buffer(buffer);
    fft->update();
}
``` 
@Haotian-Shi-cyber

**Comment by Haotian-Shi-cyber** [comment]
yeah I forgot that, you can delete this param as long as the program could run
